defaults.go

```diff
@@ -14,6 +14,13 @@ var (
 
 const (
 	fieldName = "default"
+
+	autoBase = 0
+
+	bitSize8  = 8
+	bitSize16 = 16
+	bitSize32 = 32
+	bitSize64 = 64
```

Comment:
I don't really see the point in defining these constants...

Reply:
i agree with bitXXSize constants , but autoBase add context to 0 value in base but it is not so significant therefore I will change that